src/locales/en/messages.json

```diff
@@ -3221,5 +3221,26 @@
   },
   "onlyOrgPolicyWarning": {
     "message": "Organization members who are already a member of another organization will be removed from your organization."
+  },
+  "requireSso": {
+    "message": "Single Sign-On Authentication"
+  },
+  "requireSsoPolicyDesc": {
+    "message": "Require users to log in with the Enterprise Single Sign-On method."
+  },
+  "prerequisites": {
+    "message": "Prerequisites"
+  },
+  "requireSsoCalloutMessage": {
+    "message": "The following requirements must be met before enabling this policy:"
+  },
+  "requireSsoPolicyReq": {
+    "message": "Enable the Single Organization enterprise policy."
+  },
+  "requireSsoConfigReq": {
+    "message": "Setup and enable your organization's SSO configuration. Please visit the Business Portal to complete this action."
+  },
+  "requireSsoPolicyReqError": {
+    "message": "Single Organization policy not enabled."
```

Comment:
this seems like a duplicate of `requireSsoPolicyReq`

Reply:
The message is slightly different. `requireSsoPolicyReq` is explaining that is must be enabled while the `requireSsoPolicyReqError` is stating that it is not enabled.